### ISSUE TITLE
[Secrets.yml cleanup] Parameter deletion safeguards

### ISSUE THREAD

**Issue by rmtolmach** [department-of-veterans-affairs/va.gov-team#55720]
**PROBLEM STATEMENT**
We do not want people going into the AWS paramstore and potentially breaking the running servers, by deleting paramaters in the param store.

**TASKS**
- [x] Determine LOE to disable people from deleting things from the param store
- [ ] Potentially (pending above action step) remove permissions for certain people with deleting things
- [ ] Get details of Kyle's plan to clear the param in PS and mark it for deletion.
- [ ] Implement plan 

**DEFINITION OF DONE**
- [x] Discovery complete and linked **here**
- [ ] LOE (mentioned above) recorded in the comments of this ticket
- [ ] kyle's plan implemented

**Comment by rmtolmach** [comment]
- [ ] Determine LOE to disable people from deleting things from the param store

See https://github.com/department-of-veterans-affairs/va.gov-team/issues/54219#issuecomment-1446772244. I don't think it would be too hard since _most_ people have the `dsva-vfs-developers` permissions. Whether we do in fact want to remove those permissions is not clear. By "not too hard" I mean it's a matter of removing a few lines from terraform.

**Comment by rmtolmach** [comment]
Slack thread: https://dsva.slack.com/archives/C04CYC4LMU6/p1678722545187969

Kyle's initial idea/plan:
> have a process to clear the param in ps, and mark it for deletion. This way we can be sure nothing is going to break (by clearing the value), and after a given period of time we run a cleanup job.

**Comment by RachalCassity** [comment]
This can be moved to the automation epic.

**Comment by rmtolmach** [comment]
Just wanted to note here that I created a parameter store value called `/Atemp/rebecca` to use to play around with Tags. I was going to delete it, but remembered why I had created it in the first place! So go ahead and use it for testing for this ticket, otherwise it can safely be deleted (it's not being used anywhere in code).
Link to value: https://console.amazonaws-us-gov.com/systems-manager/parameters/Atemp/rebecca/tags?region=us-gov-west-1&tab=Table
